addons/point_of_sale/static/src/app/utils/number_buffer_service.js

```diff
@@ -277,7 +277,7 @@ class NumberBuffer extends EventBus {
             }
         } else if (input === "-") {
             if (isFirstInput) {
-                this.state.buffer = "-0";
+                this.state.buffer = "firstInput";
```

Comment:
I have put -0 as it is because only - will be overwrited as shown in figure
![image](https://github.com/user-attachments/assets/b32139d7-2134-45b8-a25e-41fc91dc502a)